src/main.rs

```diff
@@ -1,10 +1,35 @@
 use std::env;
+use std::io::{self, Write};
+use std::process;
+use std::borrow::Cow;
 
-fn main() {
-    let arguments = env::args().collect::<Vec<_>>();
-    
-    match arguments.len() {
-        1 => loop { println!("y") },
-        _ => loop { println!("{}", arguments[1]) },
+const BUFFER_CAPACITY: usize = 64 * 1024;
```

Comment:
Windows can only handle buffers of 16K size, versus 64K sizes used by everyone else.